src/Sylius/Behat/Page/Admin/Product/CreateSimpleProductPage.php

```diff
@@ -163,86 +111,24 @@ protected function getDefinedElements(): array
-                'code' => '#sylius_product_code',
-                'form' => 'form[name="sylius_product"]',
-                'images' => '#sylius_product_images',
-                'language_tab' => '[data-locale="%locale%"] .title',
-                'locale_tab' => '#attributesContainer .menu [data-tab="%localeCode%"]',
-                'main_taxon' => '#sylius_product_mainTaxon',
-                'product_taxons' => '#sylius_product_productTaxons',
-                'name' => '#sylius_product_translations_%locale%_name',
-                'non_translatable_attribute_value' => '#attributesContainer [data-test-product-attribute-value-in-locale="%attributeName% "] input',
-                'original_price' => '#sylius_admin_product_variant_channelPricings_%channelCode%_originalPrice',
-                'price' => '#sylius_admin_product_variant_channelPricings_%channelCode%_price',
-                'prices_validation_message' => '#sylius_admin_product_variant_channelPricings ~ .sylius-validation-error, #sylius_admin_product_variant_channelPricings .sylius-validation-error',
-                'price_calculator' => '#sylius_admin_product_variant_pricingCalculator',
-                'shipping_category' => '#sylius_admin_product_variant_shippingCategory',
-                'shipping_required' => '#sylius_admin_product_variant_shippingRequired',
-                'slug' => '#sylius_product_translations_%locale%_slug',
-                'tab' => '.menu [data-tab="%name%"]',
-                'taxonomy' => 'a[data-tab="taxonomy"]',
-                'toggle_slug_modification_button' => '.toggle-product-slug-modification',
+                'channel' => '[data-test-channel-code="%channel_code%"]',
+                'code' => '[data-test-code]',
+                'enabled' => '[data-test-enabled]',
+                'field_shipping_category' => '[name="sylius_admin_product[variant][shippingCategory]"]',
+                'field_shipping_required' => '[name="sylius_admin_product[variant][shippingRequired]"]',
+                'form' => '[data-live-name-value="sylius_admin:product:form"]',
+                'product_translation_accordion' => '[data-test-product-translations-accordion="%localeCode%"]',
+                'side_navigation_tab' => '[data-test-side-navigation-tab="%name%"]',
             ],
-            $this->getDefinedFormElements(),
         );
     }
 
-    private function openTaxonBookmarks(): void
-    {
-        $this->getElement('taxonomy')->click();
-    }
-
-    private function selectElementFromAttributesDropdown(string $id): void
-    {
-        $this->getDriver()->executeScript('$(\'#sylius_product_attribute_choice\').dropdown(\'show\');');
-        $this->getDriver()->executeScript(sprintf('$(\'#sylius_product_attribute_choice\').dropdown(\'set selected\', \'%s\');', $id));
-    }
-
-    private function waitForFormElement(int $timeout = 5): void
+    private function waitForFormUpdate(): void
```

Comment:
This method should be extracted to the `Crud/CreatePage`

Reply:
https://github.com/Sylius/Sylius/pull/16349/files#diff-c78395e2fc4a5b43ac7314d83ba3fcc4fa7e283646ee81550733bd722c21334dR89